indico_toolkit/indico_wrapper/workflow.py

```diff
@@ -104,7 +128,11 @@ def get_submission_results_from_ids(
                 elif not return_failed_results:
                     print(message)
                     continue
+            if submission.deleted and ignore_deleted_submissions:
```

Comment:
if it is deleted, I'm guessing an exception will be raised at self._create_result- wonder if we should at least log that it is deleted (even if it's ignored), bc/ I'm guessing the exception won't explicitly state that

Reply:
What is the behavior today for deleted submissions? Would the exception be handled at all? I do agree that some info about deleted submissions makes sense, but I think that's regardless of whether the `ignore_deleted_submissions` flag is True.

I'd think that it makes sense to check if a submission is deleted prior to calling `_create_result`, if that doesn't cause any other issues.